pkgs/applications/networking/feedreaders/feeds/default.nix

```diff
@@ -0,0 +1,108 @@
+{ lib
+, stdenv
+, fetchFromGitLab
+
+, appstream
+, gobject-introspection
+, meson
+, ninja
+, pkg-config
+, wrapGAppsHook
+
+, glib
+, gtk3
+, libhandy
+, webkitgtk
+, python3
+}:
+let
+  listparser = python3.pkgs.buildPythonPackage rec {
```

Comment:
Why not add this to python-packages?

Reply:
Nothing else depends on it, and the package also hasn't been updated in a good while (last commit on the upstream repo is from September 2017). I was under (perhaps wrong) impression Python maintainers are reluctant to accept a new package like this and making it a private / vendored package was preferred.

Reply:
That makes sense. I think that's a pretty good reason for keeping it private.

Reply:
I'm packaging [gfeeds](https://gitlab.gnome.org/World/gfeeds), which is depends on listparser.